docs/helping.md

```diff
@@ -1,38 +1,55 @@
 
 "My controller isn't working": `/tag id:controller` (This links to the specific section on adding launch options/Northstar's `.exe` file to Steam to use it's controller support, as well as mentioning DS4Windows as a backup in the case of a PS4/DS4 (**D**ual**S**hock**4**) controller)
 
-"How do I play with friends?": `/tag id:host-listen` (This goes over hosting a listen server, which is what hitting `Private Match` in Northstar does. Try to inform the user that they can't invite friends normally and have to do this in order to make a private match. If they ask about port forwarding, ask them to Google how to do it for their specific router as it tends to differ, sometimes heavily, from router to router)
+"How do I play with friends?": `/tag id:host-listen` (This goes over hosting a listen server, which is what hitting `Private Match` in Northstar does. 
+Try to inform the user that they can't invite friends normally and have to do this in order to make a private match. 
+If they ask about port forwarding, ask them to Google how to do it for their specific router as it tends to differ, sometimes heavily, from router to router)
 
-"How to play co-op?": `/tag id:coop` (This gives a detailed mini explanation on how to install the co-op mod, and host a server to run it. Try to inform the user that it's still heavily WIP and has some issues. Listen servers don't allow for joining mid game; to do this, they must host a dedicated server with the "coop patches" mod provided with this tag)
+"How to play co-op?": `/tag id:coop` (This gives a detailed mini explanation on how to install the co-op mod, and host a server to run it. 
+Try to inform the user that it's still heavily WIP and has some issues. 
+Listen servers don't allow for joining mid game; to do this, they must host a dedicated server with the "coop patches" mod provided with this tag)
 
-"How to play Frontier Defnse?": `/tag id:fd` (This is called `fd` as it's the abbreviation for **F**rontier **D**efense. This tag goes over how to install and set up Frontier Defense, like co-op's tag. Try to inform the user that it's still heavily WIP)
+"How to play Frontier Defnse?": `/tag id:fd` (This is called `fd` as it's the abbreviation for **F**rontier **D**efense. 
+This tag goes over how to install and set up Frontier Defense, like co-op's tag. Try to inform the user that it's still heavily WIP)
 
 "How do I use X mod manager?": The 3 "main" mod managers all have guides. `/tag id:{INSTALLER NAME HERE}-guide` links to the specified installer's guide
 
-"How do I (manually) install Northstar?": `/tag id:manual-install` (This goes over installing, extracting, and replacing the files as well as giving a video guide. Spectre should catch _most_ of these questions)
+"How do I (manually) install Northstar?": `/tag id:manual-install` (This goes over installing, extracting, and replacing the files as well as giving a video guide. 
+Spectre should catch _most_ of these questions)
```

Comment:
Eh maybe avoid Spectre mention until it's added. Also when adding, might make sense to mention somewhere what _Spectre_ is ^^

Reply:
this may or may not have been left over from an original commit where i was assuming id be working on this for weeks and Spectre would've been added already, there was a section explaining Spectre but it's since removed

Reply:
The original explanation:
> As of {ENTER DATE HERE WHEN SPECTRE GETS ADDED}, Cyn(cooldudepugs)'s Discord bot [Spectre](https://github.com/CooldudePUGS/Spectre) was added to Northstar's Discord. This bot tries to automatically redirect users to opening a ticket in the `#help` channel, and automatically replies to some messages regarding issues with the game, however it cannot and will not catch all cases, even of basic help.

Would probably edit this to say 
> As of {ENTER DATE HERE WHEN SPECTRE GETS ADDED}, Cyn(cooldudepugs)'s Discord bot [Spectre](https://github.com/CooldudePUGS/Spectre) was added to the Northstar Discord. This bot tries to automatically redirect users to opening a ticket in the `#help` channel, and automatically replies to some messages regarding issues or questions with Northstar. It, however, cannot and will not catch all cases, even of basic help.